app/experimenter/experiments/constants/nimbus.py

```diff
@@ -409,6 +409,34 @@ class NimbusTargetingConfig:
     application_choice_names=(Application.DESKTOP.name,),
 )
 
+TARGETING_INFREQUENT_USER_NEED_DEFAULT_HAS_PIN = NimbusTargetingConfig(
```

Comment:
Right now, there are different orders in the naming (either DEFAULT or PIN can come first).  In the targeting expression, default is always first.  I'd like to propose that you put default always first in the naming too, as it requires less error-prone thinking when reviewing whether the targeting matches the naming.

A similar problem we can't directly fix here is that the one targeting variable (isDefaultBrowser) is in the positive, and the other (doesAppNeedPin) is in the negative. However, I think we can limit the error-prone-ness pretty easily by facting out and reusing TARGETING_HAS_DEFAULT_NEEDS_PIN and TARGETING_NEEDS_DEFAULT_HAS_PIN, which would cut the number of possible places for error from 8 down to two.  I'm open to debate on this second one, however.

I'd like to propose that you always 


*

Reply:
Good idea, Dan. I've taken both suggestions and updated accordingly.